cmd/install.go

```diff
@@ -52,8 +52,8 @@ var installCommand = &cobra.Command{
 			return err
 		}
 		
-		if imageInspect.Config.Entrypoint == nil {
-			return fmt.Errorf("the image '%s' is not compatible with Whalebrew: it does not have an entrypoint", imageName)
+		if imageInspect.Config.Entrypoint == nil && customEntrypoint == "" {
+			return fmt.Errorf("The image '%s' does not have an entrypoint, please provide a custom entrypoint", imageName)
```

Comment:
Shall we inform how to do so?
```suggestion
			return fmt.Errorf("The image '%s' does not have an entrypoint, please provide a custom entrypoint adding '--entrypoint my-program'", imageName)
```